api/handlers/find_user.go

```diff
@@ -16,21 +15,28 @@ type findUser struct{
 	rt *ShivaniCustomServerExample1.Runtime
 }
 func (f *findUser) Handle(fup users.FindUsersParams) middleware.Responder{
-	// fup.Name
-	// v:=service.NewManager()
-	// v.ListUsers()
-	
-	fmt.Println("Application Name is",f.rt.GetApplicationName())
-	// n:="Shivani Sharma"
-	// us:= []*models.User{{Address:"ABC",ID:2,Name:&n}}
-	
-	us,_:= f.rt.GetManager().ListUsers(fup.Name,fup.Limit)
+
+	var limit int32
+	// var name string
+	var us []*domain.User
+	// if fup.Limit != nil && fup.Name != nil {
+		limit = *fup.Limit
+		// name = *fup.Name
+		us,_ = f.rt.GetManager().ListUsers(filteredMap(fup.Name), limit)
```

Comment:
need to handle err as well 
it should be looks 
`us, err :== f.rt.GetManager().ListUsers(filteredMap(fup.Name), limit)`

Reply:
Done ! Please check